src/zui/components/ZUIButton/index.tsx

```diff
@@ -59,21 +59,25 @@ const getTextPadding = (
 ) => {
   if (size === 'large') {
     if (variant === 'secondary') {
-      return '0.438rem 1.375rem 0.438rem 1.375rem';
+      return '0.467rem 1.375rem';
```

Comment:
Was `ZUIButton` wrong before?

Reply:
Yep. It never got properly reviewed, it seems, since it was built as part of the header - the first thing I did back in august.